docs/content/mobile/android/local-deploy.md

```diff
@@ -16,12 +16,18 @@ To connect your Android device to a local deployment, you need to prepare its co
 1. In the **Runtime IP for local deployment** field, enter your local machine's IP address. (You can find how to do this on the [Lifehacker website](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address).)
 1. Select **Save** and close the window.
 
-Create and add a MonoBehaviour script to your Android client-worker GameObject. To do this:
+<%(Callout message="If you are using one of our Starter Projects, you can skip the **Create a mobile connector script** section below, as you already have one in your project.")%>
 
-1. If your project does not already contain one, create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs) and include the functionality you want.
+<%(#Expandable title="Create a mobile connector script")%>
+
+If you [added the GDK]({{urlRoot}}/content/set-up-new-project) to an existing Unity project rather than using a Starter Project, then you also need to create and add a MonoBehaviour script to your Android client-worker GameObject. To do this:
+
+1. If your project does not already contain one, create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs) and include the functionality you want. You can base your implementation on [the one](https://github.com/spatialos/gdk-for-unity-fps-starter-project/blob/master/workers/unity/Assets/Fps/Scripts/SetupLogic/AndroidWorkerConnector.cs) in our FPS Starter Project.
```

Comment:
after looking at the code a bit: it might be better to use the one in the Blank project as an example as the `AndroidWorkerConnector` contains functionality that is specific to the FPS starter project. 

I also think we can remove the first part `If your project does not already contain one,` because we already assume that it's not a Spatial project

Reply:
Thanks @jessicafalk. I've cited the ClientWorkerConnector.cs script from the [Blank Project](https://github.com/spatialos/gdk-for-unity-blank-project/blob/master/workers/unity/Assets/Scripts/Workers/AndroidClientWorkerConnector.cs) and removed
>If your project does not already contain one